scripts/_util.py

```diff
@@ -27,3 +36,33 @@ def validate_bytes(
         raise click.BadParameter("not a hex string")
 
     return decode_hex(value)
+
+
+def _account_from_keyfile(keyfile: Path, password: str) -> LocalAccount:
+    with open(keyfile, "rt") as fp:
+        privkey = Account.decrypt(json.load(fp), password)
+    return cast(LocalAccount, Account.from_key(privkey))
```

Comment:
I would actually make this public, move it to `beamer.util`, and make use of it both under `scripts/` and in `beamer/cli.py`, which already has `_account_from_keyfile`. It is a generally useful function.